config/nav.yml

```diff
@@ -296,7 +296,6 @@ nav:
         - About Eventing features: eventing/features/README.md
         - DeliverySpec.Timeout field: eventing/features/delivery-timeout.md
         - DeliverySpec.RetryAfterMax field: eventing/features/delivery-retryafter.md
-        - New Trigger Filters: eventing/features/new-trigger-filters.md
```

Comment:
We need to remove that file `eventing/features/new-trigger-filters.md`

Reply:
When I removed the file, it broke links in the old release note pages. So, I decided to:
1. leave it in
2. remove it from the nav so that you can only access it through the old release notes

WDYT @pierDipi ?

Reply:
or we could update the old release note pages to point to the new place?

Reply:
Leaving it there means we need to update/maintain two places for this and it's not really worth it

Reply:
Should be fixed now